tests/Custom/SelectTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
```

Comment:
That is the point. All drivers should work the same way. It's not a bug in Selenium2 driver, that it's `setValue` implementation doesn't support selecting option by it's text. It's how it works consistently for all drivers.